### DISCUSSION TITLE
Account restricted for violation of terms of service

### DISCUSSION THREAD

**Discussion by AntonVarlamov** [codewars/codewars.com#3063] [General]
My account was restricted and my entire honor was reset due to plagiarism. This is completely my fault and I admit it, but is it possible to remove it, I just started learning go and I would like to see other people's solutions what approaches and packages they use to develop.
P.S. If something is not clear, English is not my native language.

**Comment by Blind4Basics** [comment]
Hello,

I'd suggest you delete the current account and start fresh (you can use the same username for your new account if you delete the current one first). Would that "do it" for you?
